### PR TITLE
[BUGFIX] Fixed calculation of addon prices

### PR DESCRIPTION
**Pull request by turbo-ele**

### CODE REVIEW

File: Manager/PriceCalculationManager.php [hunk 1]
```diff
@@ -67,7 +69,7 @@ public function retrieveItemPrices($itemsData, $currency, $locale)
             $itemData = $this->unsetUneccesaryData($itemData);
 
             // Generate item.
-            $item = $this->getItemManager()->save($itemData, $locale);
+            $item = $this->getItemManager()->save($itemData, $locale, null, null, null, null, $previousItem);
```
Comment (trickreich) [inline Manager/PriceCalculationManager.php:72]: not nice, but ok

**Comment by trickreich** [comment]
ACK

**Comment by trickreich** [comment]
@turbo-ele but tests are failing

**Comment by turbo-ele** [comment]
yes, but they weren't before. This should probably fixed some time ;)

**Pull request merged**